docs/source/distributions.md

```diff
@@ -1,460 +1,690 @@
 
+```{eval-rst}
 .. currentmodule:: torch.distributions.geometric
+```
+
+```{eval-rst}
 .. autoclass:: Geometric
     :members:
     :undoc-members:
     :show-inheritance:
+```
 
-:hidden:`Gumbel`
-~~~~~~~~~~~~~~~~
+## {hidden}`Gumbel`
 
+```{eval-rst}
 .. currentmodule:: torch.distributions.gumbel
+```
+
+```{eval-rst}
 .. autoclass:: Gumbel
     :members:
     :undoc-members:
     :show-inheritance:
+```
 
-:hidden:`HalfCauchy`
-~~~~~~~~~~~~~~~~~~~~~~~
+## {hidden}`HalfCauchy`
 
+```{eval-rst}
 .. currentmodule:: torch.distributions.half_cauchy
+```
+
+```{eval-rst}
 .. autoclass:: HalfCauchy
     :members:
     :undoc-members:
     :show-inheritance:
+```
 
-:hidden:`HalfNormal`
-~~~~~~~~~~~~~~~~~~~~~~~
+## {hidden}`HalfNormal`
 
+```{eval-rst}
 .. currentmodule:: torch.distributions.half_normal
+```
+
+```{eval-rst}
 .. autoclass:: HalfNormal
     :members:
     :undoc-members:
     :show-inheritance:
+```
 
-:hidden:`Independent`
-~~~~~~~~~~~~~~~~~~~~~
+## {hidden}`Independent`
 
+```{eval-rst}
 .. currentmodule:: torch.distributions.independent
+```
+
+```{eval-rst}
 .. autoclass:: Independent
     :members:
     :undoc-members:
     :show-inheritance:
+```
 
-:hidden:`InverseGamma`
-~~~~~~~~~~~~~~~~~~~~~~
+## {hidden}`InverseGamma`
 
+```{eval-rst}
 .. currentmodule:: torch.distributions.inverse_gamma
+```
+
+```{eval-rst}
 .. autoclass:: InverseGamma
     :members:
     :undoc-members:
     :show-inheritance:
+```
 
-:hidden:`Kumaraswamy`
-~~~~~~~~~~~~~~~~~~~~~
+## {hidden}`Kumaraswamy`
 
+```{eval-rst}
 .. currentmodule:: torch.distributions.kumaraswamy
+```
+
+```{eval-rst}
 .. autoclass:: Kumaraswamy
     :members:
     :undoc-members:
     :show-inheritance:
+```
 
-:hidden:`LKJCholesky`
-~~~~~~~~~~~~~~~~~~~~~~~
+## {hidden}`LKJCholesky`
 
+```{eval-rst}
 .. currentmodule:: torch.distributions.lkj_cholesky
+```
+
+```{eval-rst}
 .. autoclass:: LKJCholesky
     :members:
     :undoc-members:
     :show-inheritance:
+```
 
-:hidden:`Laplace`
-~~~~~~~~~~~~~~~~~~~~~~~
+## {hidden}`Laplace`
 
+```{eval-rst}
 .. currentmodule:: torch.distributions.laplace
+```
+
+```{eval-rst}
 .. autoclass:: Laplace
     :members:
     :undoc-members:
     :show-inheritance:
+```
 
-:hidden:`LogNormal`
-~~~~~~~~~~~~~~~~~~~~~~~
+## {hidden}`LogNormal`
 
+```{eval-rst}
 .. currentmodule:: torch.distributions.log_normal
+```
+
+```{eval-rst}
 .. autoclass:: LogNormal
     :members:
     :undoc-members:
     :show-inheritance:
+```
 
-:hidden:`LowRankMultivariateNormal`
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+## {hidden}`LowRankMultivariateNormal`
 
+```{eval-rst}
 .. currentmodule:: torch.distributions.lowrank_multivariate_normal
+```
+
+```{eval-rst}
 .. autoclass:: LowRankMultivariateNormal
     :members:
     :undoc-members:
     :show-inheritance:
+```
 
-:hidden:`MixtureSameFamily`
-~~~~~~~~~~~~~~~~~~~~~~~~~~~
+## {hidden}`MixtureSameFamily`
 
+```{eval-rst}
 .. currentmodule:: torch.distributions.mixture_same_family
+```
+
+```{eval-rst}
```

Comment:
this should fix build [error](https://github.com/pytorch/pytorch/actions/runs/15549382341/job/43834218730?pr=155297#step:11:1075).
```suggestion

```